src/main/java/org/repodriller/scm/SCM.java

```diff
@@ -68,7 +69,7 @@
 	Commit getCommit(String id);
 	/* TODO A method named getCommitXYZ should return a Commit. */
 	String getCommitFromTag(String tag);
-
+	List<Modification> getDiffBetweenCommits(String priorCommit, String laterCommit);
```

Comment:
Javadoc. And does it matter whether A -> B or B -> A in the history?

Reply:
Yeah, it does. Adding Javadoc that says this and also throwing an IllegalArgumentException if this assumption is not met.

Reply:
Well, actually it looks like it doesn't. [DiffFormatter](http://download.eclipse.org/jgit/docs/jgit-2.0.0.201206130900-r/apidocs/org/eclipse/jgit/diff/DiffFormatter.html#scan(org.eclipse.jgit.lib.AnyObjectId, org.eclipse.jgit.lib.AnyObjectId)) don't mention anything. I guess I shouldn't go against the normal git operation, which allows the 'wrong' order of commits.